.github/dependabot.yml

```
@@ -36,6 +36,10 @@ updates:
       - /providers/fab/src/airflow/providers/fab/www
     schedule:
       interval: daily
+    groups:
+      ui-package-updates:
```

```suggestion
      fab-package-updates:
```

So that provider PRs are separate to ones touching "core"

---

Oh wait, that won't work.

Why do we have this twice?

---

Oh, 2.x and main, I see.

---

yes its 2.x